modules/osdk-updating-projects.adoc

```diff
@@ -0,0 +1,353 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/golang/osdk-golang-updating-projects.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-updating-projects.adoc
+// * operators/operator_sdk/helm/osdk-helm-updating-projects.adoc
+// * operators/operator_sdk/helm/osdk-hybrid-helm-updating-projects.adoc
+
+ifeval::["{context}" == "osdk-golang-updating-projects"]
+:golang:
+:type: Go
+endif::[]
+ifeval::["{context}" == "osdk-ansible-updating-projects"]
+:ansible:
+:type: Ansible
+endif::[]
+ifeval::["{context}" == "osdk-helm-updating-projects"]
+:helm:
+:type: Helm
+endif::[]
+ifeval::["{context}" == "osdk-hybrid-helm-updating-projects"]
+:hybrid:
+:type: Hybrid Helm
+endif::[]
+
+:osdk_ver: 1.22.0
+:osdk_ver_n1: 1.16.0
+
+:_content-type: PROCEDURE
+[id="osdk-upgrading-projects_{context}"]
+= Updating {type}-based Operator projects for Operator SDK {osdk_ver}
+
+The following procedure updates an existing {type}-based Operator project for compatibility with {osdk_ver}.
+
+.Prerequisites
+
+* Operator SDK {osdk_ver} installed.
+* An Operator project created or maintained with Operator SDK {osdk_ver_n1}.
+
+.Procedure
+
+. Make the following changes to the `config/default/manager_auth_proxy_patch.yaml` file: 
++
+[source,yaml]
+----
+...
+spec:
+  template:
+    spec:
+      containers:
+      - name: kube-rbac-proxy
+        image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.11 <1>
+        args:
+        - "--secure-listen-address=0.0.0.0:8443"
+        - "--upstream=http://127.0.0.1:8080/"
+        - "--logtostderr=true"
+        - "--v=0" <2>
+...
+resources:
+  limits:
+    cpu: 500m
+    memory: 128Mi
+  requests:
+    cpu: 5m
+    memory: 64Mi <3>
+----
+<1> Update the tag version from `v4.10` to `v4.11.`.
+<2> Reduce the debugging log level from `"--v10"` to `"--v=0"`.
+<3> Add resource requests and limits.
+
+. Make the following changes to your `Makefile`:
+
+.. Enable support for image digests by adding the following environment variables:
++
+.Old `Makefile`
+[source,terminal]
+----
+BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
+...
+----
++
+.New `Makefile`
+[source,terminal]
+----
+BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
+
+# BUNDLE_GEN_FLAGS are the flags passed to the operator-sdk generate bundle command
+BUNDLE_GEN_FLAGS ?= -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+
+# USE_IMAGE_DIGESTS defines if images are resolved via tags or digests
+# You can enable this value if you would like to use SHA Based Digests
+# To enable set flag to true
+USE_IMAGE_DIGESTS ?= false
+ifeq ($(USE_IMAGE_DIGESTS), true)
+	BUNDLE_GEN_FLAGS += --use-image-digests
+endif
+----
+
+.. Replace the bundle target with the `BUNDLE_GEN_FLAGS` environment variable:
++
+.Old `Makefile`
+[source,terminal]
+----
+$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
```

Comment:
It is probably my lack of knowledge with OSDK, but is this a command? Like the first step was that you were manually editing the makefile, and now you're running some command? Does that update something in the makefile?

Maybe it's the "Replace" wording that's throwing me off, thinking that I'm editing a file and making a replacement.

It also could be the titles of the code blocks in this step 2b. Are these really the old and new makefile, or should these headings be something different?

Reply:
There isn't a source time for Makefiles, so the only formatting option is `source,terminal`. And the filename and the filetype are both `Makefile`

Reply:
@michaelryanpeter For generic files, you can use [source,text]